File: 20221128/MaxAndMin/src/main/java/MaxAndMin.java
class MaxAndMin {
    public String solution(String s) {
        String[] numbers = s.split(" ");

        int max = Integer.parseInt(numbers[0]);
        int min = Integer.parseInt(numbers[0]);

        for (String number : numbers) {
            max = Math.max(max, Integer.parseInt(number));
            min = Math.min(min, Integer.parseInt(number));
        }

        String answer = min + " " + max;
        return answer;
    }
}

File: 20221129/MaxAndMin.js
export default function solution(s) {
  const numbers = s.split(' ');

  const answer = [numbers[0], numbers[0]];

  numbers.forEach((element) => {
    answer[0] = Math.min(answer[0], element);
    answer[1] = Math.max(answer[1], element);
  });

  return answer.join(' ');
}
